WTWSMS/common/religions/00_religions.txt

```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```

Comment:
"So you would say neither polygamy nor concubinage?"

Definitely. The Christian "lovers" system is absolutely perfect.